Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -4,19 +4,26 @@ A [Leap Motion Controller](https://www.ultraleap.com/product/leap-motion-control
 
 The Leap Motion Data Provider enables articulated hand tracking for VR and could be useful for rapid prototyping in the editor.  The data provider can be configured to use the leap motion controller mounted on a headset or placed on a desk face up.
 
+![LeapMotionIntroGif](../Images/CrossPlatform/LeapMotion/LeapHandsGif3.gif)
+
 ## Using Leap Motion tracking in MRTK
 1. Prepare MRTK project for Leap Motion
 
-    - **This step only applies if the source of MRTK is cloned from the git repo and NOT the from the Unity packages. If the MRTK source is from the Unity packages, go to step 2.**
+    - **This step only applies for the Leap Motion Core Assets version 4.4.0 and if the source of MRTK is cloned from the git repo, NOT from the Unity packages. This step is not required if the Leap Motion Core Assets version 4.5.0 are used. If the MRTK source is going to be from the Unity packages, start at step 2.**
 
     - Navigate to **Mixed Reality Toolkit > Utilities > Leap Motion > Configure CSC File for Leap Motion**. Updating the csc file filters out the obsolete warnings produced by the Leap Motion Core Assets.  The MRTK repo contains a csc file that converts warnings to errors, this conversion halts the Leap Motion MRTK configuration process.  The obsolete warnings issue is tracked [here](https://github.com/leapmotion/UnityModules/issues/1082).
 
     ![LeapMotionUpdateCSCFile](../Images/CrossPlatform/LeapMotion/LeapMotionConfigureCSCFile2.png)
 
-1. Importing the Leap Motion Core Assets
-    - Import the **Microsoft.MixedReality.Toolkit.Foundation** package into the unity project.
+1. Importing MRTK and the Leap Motion Core Assets
+    - Import the **Microsoft.MixedReality.Toolkit.Foundation** package into the Unity project.
+    > [!NOTE]
+    > The Leap Motion Data Provider is supported in both Unity 2018.4.x and 2019.3.x, but 2019.3.x is preferred.
```

Comment:
Is this comment something that we back, or is that something that LeapMotion recommends? If it does work in both it might be safer to just not say anything specific here (or link to the leap page about recommended configurations). Basically I don't want to duplicate recommendations (and have them potentially get out of sync in the future)

Reply:
The preference for 2019.3 comes from my experience adding and removing the leap core assets on both versions of Unity.  Specifically the removal of the leap core assets in 2018 require an extra unity restart.  But it can be removed.